src/test/java/nl/tudelft/semgroup4/GameStateControllerTest.java

```diff
@@ -0,0 +1,139 @@
+package nl.tudelft.semgroup4;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import nl.tudelft.model.Countdown;
+import nl.tudelft.model.Game;
+import nl.tudelft.semgroup4.resources.ResourcesWrapper;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito;
+import org.newdawn.slick.GameContainer;
+import org.newdawn.slick.Graphics;
+import org.newdawn.slick.Input;
+import org.newdawn.slick.SlickException;
+import org.newdawn.slick.Sound;
+import org.newdawn.slick.state.StateBasedGame;
+
+public class GameStateControllerTest {
+    
+    private GameStateController gsController;
+    private Game mockedGame;
+    
+    @Before
```

Comment:
Usually Checkstyle requires a comment on @Before methods. Did you specifically suppress this, cause in that case we might want to apply this to all tests.

otherwise: add a JavaDoc to not cause a Checkstyle violation.

< documentation, configuration.